group_vars/all

```diff
@@ -101,6 +101,12 @@ pkg_version_hlacon_conda_pyqt: "5.12.3"
 pkg_version_hlacon_conda_pydm: "1.10.4"
 pkg_version_hlacon_conda_siriushlacon: "2.0.13"
 
+# === fac docker services ===
+
+pkg_version_docker_machine_applications: "master"
```

Comment:
@xresende, why are we using the master branch here?

Reply:
at its final version, these repos are stateless: they simply take states (pkg_versions) defined in ansible configs and apply to docker images, recreating new versions to be deployed. I think adding versioning to these repos is not natural and only adds maintenance and elaboration into the deploy process.

Reply:
at the end, the way I devised the deploy process, everything will be managed from ansible makefile targets: deploy into desktops, IOC and csconsts docker images, and ansible deploy tag creation.